.github/workflows/SQL2019.yaml

```diff
@@ -0,0 +1,197 @@
+
+      - name: Create Logins for Test DBs
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          CREATE LOGIN AirlineUser WITH PASSWORD = '%dbPassword%', CHECK_POLICY=OFF, CHECK_EXPIRATION = OFF, DEFAULT_DATABASE=AirlineTestDB;
+          CREATE LOGIN AirlineUserdbowner WITH PASSWORD = '%dbPassword%', CHECK_POLICY=OFF, CHECK_EXPIRATION = OFF, DEFAULT_DATABASE=AirlineTestDB;"
+      
+      - name: Alter AirlineUser
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          USE AirlineTestDB;
+          ALTER USER [AirlineUser] WITH LOGIN=[AirlineUser]"
+      
+      - name: Alter AirlineUserdbowner
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          USE AirlineTestDB;
+          ALTER USER [AirlineUserdbowner] WITH LOGIN=[AirlineUserdbowner]"
+
+      - name: Alter Authorization
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          USE AirlineTestDB;
+          ALTER AUTHORIZATION ON SCHEMA::[db_owner] TO [AirlineUserdbowner]"
+
+      - name: Set up R ${{ env.r-version }} Runtime
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: ${{ env.r-version }}
+
+      # Todo: Needs mirror installation URL for python 3.5
+      - name: Install R Package Dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          cache-version: 2
+          working-directory: ./R
+          extra-packages:
+            #Retrieves most recent odbc pkg from cran to avoid errors seen in older versions.
+            #Current version needs R >=3.6 and is currently failing
+            cran::odbc
+            cran::xml2
+            rcmdcheck
+        continue-on-error: true
+
+      - uses: r-lib/actions/check-r-package@v2
+        with:
+          working-directory: ./R
+        env:
+          PASSWORD_AIRLINE_USER: "${{ env.dbPassword }}"
+          PASSWORD_AIRLINE_USER_DBOWNER: "${{ env.dbPassword }}"
+        continue-on-error: true
+
+
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.python-version }}
+
+      - name: Install Python dependencies
+        working-directory: ./Python
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install flake8 pytest
+          pip install -r requirements.txt
+
+      - name: Build Python Package
+        working-directory: ./Python
+        run: ./buildandinstall.cmd
+
+      - name: Run pytest
+        working-directory: ./Python/tests
+        run: |
+          pytest
+        continue-on-error: true
```

Comment:
add newline to get rid of this github warning:
![image](https://github.com/microsoft/sqlmlutils/assets/6414189/caed1aa7-4058-4820-b878-0674285a6c4a)
context i found that explains why newline is needed. Other SO articles explain the github icon
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log/5813359#5813359